projects/Mallard/src/components/button/button.tsx

```diff
@@ -21,17 +23,28 @@ export enum ButtonAppearance {
     apricot,
 }
 
+const height =
```

Comment:
probably not. I normally prefer using padding and letting the height calculate itself but since we have round buttons (the ones with icons) and they use a different font (the icon font), hardcoding the button height lets us make super sure the circle buttons are perfect circles!

That's also why I'm accounting for the font scaling manually here, so the icon will fit as it will scale ith the rest of the fonts